src/components/marker.js

```diff
@@ -61,9 +61,10 @@ export default class Marker extends Component {
     };
 
     return createElement('div', {
-      className: 'mapboxgl-marker',
+      className: className,
       style: containerStyle,
-      children: this.props.children
+      children: this.props.children,
+      onClick: onClick
```

Comment:
This seems arbitrary. Any reason you cannot register the `onClick` event on the child component?

Reply:
Just wanted to make it clearly.